test/test.js

```diff
@@ -189,3 +189,15 @@ test('Should sort prefixed propertyes as unprefixed if first one not in order, b
         'width'
     ] });
 });
+
+test('Should insert empty lines between children classes in accordance with option \'empty-lines-between-children-rules\'', t => {
+    return run(t, 'prefixed', {
```

Comment:
Should be `return run(t, 'lines-between-children.expected', {`, because second argument for `run()` is file name without extension.